doc/tutorials/lris_howto.rst

```diff
@@ -0,0 +1,485 @@
+
+.. include:: ../include/links.rst
+
+.. _lris_howto:
+
+=================
+Keck-LRIS HOWTO
+=================
+
+Overview
+========
+
+This doc goes through a full run of PypeIt on one of our :ref:`lris`
+datasets, specifically the ``keck_lris_red_mark4/multi_600_10000_slitmask``
+dataset. These are Keck LRIS RED multi-slit observations taken with the new
+Mark IV detector and the 600/10000 grating. See :ref:`here <dev-suite>`
+to find the example dataset.
+
+If you're having trouble reducing your data, we encourage you to try going
+through this tutorial using this example dataset first. Please join our
+`PypeIt Users Slack <pypeit-users.slack.com>`__ (using
+`this invitation link
+<https://join.slack.com/t/pypeit-users/shared_invite/zt-1kc4rxhsj-vKU1JnUA~8PZE~tPlu~aTg>`__)
+to ask for help, and/or `Submit an issue`_ to Github if you find a bug!
+
+The following was performed on a Macbook Pro with 16 GB RAM and took approximately 30 minutes.
+
+
+Setup
+=====
+
+Organize data
+-------------
+
+Identify the folder where the raw data are stored and make sure you have
+all the calibration files you need, in addition to the science ones.
+In this example, the raw data are stored in the folder
+``/PypeIt-development-suite/RAW_DATA/keck_lris_red_mark4/multi_600_10000_slitmask``.
+
+The files within this folder are:
+
+.. code-block:: bash
+
+    $ ls -1
+    r230417_01033.fits
+    r230417_01064.fits
+    r230417_01072_with_maskinfo.fits
+
+This folder can include data from different datasets (e.g., more than one slitmask
+or observations in various gratings). The script :ref:`pypeit_setup`
+(see next step) will help to parse the desired dataset.
+
+.. note::
+
+        Pypeit is able to incorporate slitmask information in the reduction routine,
```

Comment:
We should check with WMKO about archived slitmask files.  I think there are more out there than the common person knows.

Reply:
Yes, we can ask. In the meantime I added some info on what those slitmask files are.

Reply:
good good